content/docs/CLI/local-build.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "Local Build"
+date: 2021-03-18T08:52:33+01:00
+draft: false
+---
+
+# local-build
+
+Create a local build for the content of the upstream repository.
```

Comment:
I don't like this "local build for ... ". Can we simply copy the line from help, i.e.
"Create RPMs using content of the upstream repository." ?